src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -17,6 +18,8 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
       case x => x.toString.toLowerCase
     }
 
+    val timestamp = new SimpleDateFormat("YYYY/MM/dd HH:mm:ss:SS").format(new Date())
```

Comment:
Any reason not to be consistent with the server date format?

`2015-10-20 12:02:49,318`